machine/storage/backends/sqlite.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import aiosqlite
+import time
+from machine.storage.backends.base import MachineBaseStorage
+from typing import Any, Mapping
+from contextlib import AsyncExitStack
+
+
+class SQLiteStorage(MachineBaseStorage):
+    _context_stack: AsyncExitStack
+
+    def __init__(self, settings: Mapping[str, Any]):
+        self._file = settings.get("SQLITE_FILE", "slack-machine-state.db")
+
+    async def close(self) -> None:
+        await self.conn.close()
+
+    async def init(self) -> None:
+        self.conn = await aiosqlite.connect(self._file)
+        self.conn.text_factory = bytes
+        self.cursor = await self.conn.cursor()
+        await self.cursor.execute("""
+            CREATE TABLE IF NOT EXISTS storage (
+                key TEXT PRIMARY KEY,
+                value TEXT NOT NULL,
+                expires INTEGER
+            )
+        """)
+        await self.conn.commit()
+
+    async def set(self, key: str, value: bytes, expires: int | None = None) -> None:
+        await self.cursor.execute(
+            """
+            INSERT OR REPLACE INTO storage (key, value, expires)
+            VALUES (?, ?, ?)
+        """,
+            (key, value, expires),
+        )
+        await self.conn.commit()
+
+    async def get(self, key: str) -> bytes | None:
+        current_GMT = int(time.time())
+        await self.cursor.execute(
+            "SELECT value FROM storage WHERE key=? and (expires > ? OR expires IS NULL)", (key, current_GMT)
```

Comment:
`expires > ?` will realistically never hold true as `expires` in the `set(...)` method is the number of seconds from now when the key expires.

Example: 

```python
# This sets a key to be expired 300 seconds from now (i.e. 5 minutes)
set("my-key", b"my-value", 300)`
```

This means that in the `get(...)` method, the comparison that is made is `300 > 1691232468` (the latter is the current epoch time gotten from `time.time()`) which will not be true, even if the key was just set.

The change you need to make, is in the `set(...)` method set the `expires` column to `int(time.time()) + expires`. That will make the `get(...)` method work properly.